cfgov/jinja2/v1/_includes/molecules/featured-menu-content.html

```diff
@@ -22,34 +19,25 @@
 
    value.body:          Body text.
 
-   value.image:         A tuple of image info, containing:
-
-   value.image.src:     A string for the image source URI.
-
-   value.image.alt:     A string for the image alt text.
+   value.image_src:     A string for the image source URI.
 
    ========================================================================== #}
 
 {% macro render( value ) %}
-<aside class="m-featured-menu-content
-              {{ 'm-featured-menu-content__horizontal'
-                 if value.is_horizontal == true else '' }}">
-    <div class="m-featured-menu-content_text">
-        <p class="h4">
-            <a href="{{ value.link.url }}">
-                  {{ value.link.text }}
-            </a>
-        </p>
-        <p>
-            {{ value.body }}
-        </p>
-    </div>
-    <a href="{{ value.link.url }}"
-       class="m-featured-menu-content_img">
-        <img src="{{ static(value.image.src) }}"
-             alt="{{ value.image.alt }}"
-             height="150"
-             width="270">
-    </a>
+<aside class="m-featured-menu-content">
+    <p class="h4">
+        <a href="{{ value.link.url }}">
+            <img src="{{ static(value.image_src) }}"
+                 alt=""
+                 height="150"
+                 width="270">
+            <span class="m-featured-menu-content_text">
+                {{ value.link.text }}
+            </span>
+        </a>
+    </p>
```

Comment:
Sorry, one more thing. Should this text be in a heading element, not a paragraph? It's a little confusing to me because it's within the navigation. I realize I'm not sure what the right way to handle headings in complex navigation is. Presumably we'd want a screen reader to read things like the "Money topics" and "Guides" headings as one tabs through the navigation, but I'm not sure if they will be, or if it just skips to links. Something to test, I suppose, but we're venturing into out-of-scope territory.

If you think we should go ahead and make it a heading here, though, I don't think a heading can contain an image, but you could probably do:

```
aside
    a
        img
        h4
    p
```

If not, though, I might suggest restructuring it that way, anyway, so you can eliminate the wrapping `p.h4` and just use a `div class="h4"` around the `{{ value.link.text }}`.

Reply:
Could I use `<h4>` instead of `<div class="h4">`? Either looks like it would eliminate the `_text` modifier.

Reply:
That was my initial suggestion :) I only started waffling because I realized I was unsure about best practices (regarding accessibility) for using headings within complex navigation.

Reply:
Ahh right "heading element" got it. I did read it! I swear! I'll set it to the div for now as that's the safer of the two.